src/Pact/Types/Capability.hs

```diff
@@ -109,6 +109,7 @@ instance Pretty CapScope where pretty = viaShow
 -- | Runtime storage of acquired or managed capability.
 data CapSlot c = CapSlot
   { _csScope :: CapScope
+  , _csEvaluating :: Bool
```

Comment:
I would rather we not do this. Why not just restrict `enforce-verified` to a defcap body a la `compose-capability`?

Reply:
Done, I think it's nicer in the end